docs/machine-learning/what-is-mldotnet.md

```diff
@@ -0,0 +1,31 @@
+---
+title: What is ML.NET? 
+description: Learn about ML.NET, a free, open-source, and cross-platform machine learning framework that enables you to build custom AI solutions and integrate them into your .NET applications.
+author: cjgronlund
+ms.topic: overview
+ms.date: 11/06/2018
+---
+# What is ML.NET?
+
+ML.NET is a free, open-source, and cross-platform machine learning framework that enables you to build custom machine learning solutions and integrate them into your .NET applications. With the ML.NET APIs you can incorporate AI into your apps using the .NET skills you already have and without leaving the .NET framework.
```

Comment:
"without leaving the .NET framework" => "without leaving .NET".  (Framework implies this only works on .NET Framework, not .NET Core)

Reply:
+1